aws/vault-server-on-eks/configure-vault.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+set -e
+
+# INSTALLING VAULT
+echo "1. Installing vault..."
+curl -sL https://releases.hashicorp.com/vault/1.11.1/vault_1.11.1_linux_amd64.zip -o vault1.zip
```

Comment:
why this version?
this is what we work with before?

Reply:
No special reason. 
Yes.